lib/AArch64Base.ml

```diff
@@ -296,6 +300,10 @@ let inverse_cond = function
 type op = ADD | ADDS | SUB | SUBS | AND | ANDS | ORR | EOR
 type variant = V32 | V64
 
+let pp_variant = function
+  | V32 -> "V32"
+  | V64 -> "V64"
+
```

Comment:
Is this function useful ?

Reply:
it is used in the error message in a later patch for `I_OP3`